examples/advanced/quantum_volume.py

```diff
@@ -151,19 +247,36 @@ def main(num_qubits: int, depth: int, num_repetitions: int, seed: int):
         num_qubits: The number of qubits for the circuit.
         depth: The number of gate layers to generate.
         num_repetitions: The number of times to run the algorithm.
+        samplers: The samplers to run the algorithm on.
+        deconstruct: An optional function to deconstruct the model circuit's
+            gates down to the targer devices gate set.
+        optimize: An optional function to optimize the circuit's gates
+            after routing.
     """
-    for _ in range(num_repetitions):
+    result = QuantumVolumeResult()
+    for repetition in range(num_repetitions):
+        print(f"Repetition {repetition + 1}")
         model_circuit = generate_model_circuit(
             num_qubits, depth, random_state=np.random.RandomState(seed))
```

Comment:
eek! bad seeding. Create one random state at the start. Every circuit will be the same otherwise

Reply:
Done.